uk.co.compucorp.civicrm.tasksassignments/views/modal/document.html

```diff
@@ -9,7 +10,8 @@ <h4 class="modal-title" ng-switch="documentModal.isRole('admin')">
       <span ng-switch-when="false">{{ documentModal.getDocumentType(documentModal.document.activity_type_id) || 'Select Document Type' }}</span>
     </h4>
   </div>
-  <div class="modal-body" nv-file-drop uploader="documentModal.uploader">
+  <crm-loading ng-if="isLoading"></crm-loading>
```

Comment:
the `<crm-loading>` directive should *wrap* the content that it's supposed to hide. And also it has a `show` parameter that you should use to decide when to hide the spinner.
```html
<crm-loading show="!isLoading">
  <div class="modal-body" nv-file-drop uploader="documentModal.uploader">
  </div>
  <div class="modal-footer">
  </div>
</crm-loading>
```